sokutei.h

```diff
@@ -0,0 +1,72 @@
+#ifndef SOKUTEI_BENCHMARK
+#define SOKUTEI_BENCHMARK
```

Comment:
In the case of include guards, it's generally a better practice to end the name of the macro definition with the `_H` postfix. Thus, `SOKUTEI_BENCHMARK_H` in this case. There are two main reasons for that:

  * The `_H` postifx clearly indicates that this macro is part of the include guard and that's it, no other responsibility.
  * `SOKUTEI_BENCHMARK` can be an actual macro with some proper use case. On the other hand, it's hard to imagine a role for `SOKUTEI_BENCHMARK_H` other than being part of an include guard.